modules/hosted-che/partials/proc_providing-a-devfile.adoc

```diff
@@ -5,7 +5,7 @@
 [id="providing-a-devfile_{context}"]
 = Providing a devfile
 
-Providing a devfile in the root of the repository is recommended to define the development environment of the workspace created by the factory URL.
+Providing a link:https://devfile.io/[devfile] in the root of the repository is recommended to define the development environment of the workspace created by the factory URL.
 This ensures that the workspace contains everything users need to effectively review the pull request, such as plug-ins, development commands, and other environment setup.
```

Comment:
In this way, the workspace contains everything users need to review pull requests, such as plug-ins, development commands, and other environment setup.